Bundle/GoogleTests.mm

```diff
@@ -172,11 +178,15 @@ @implementation GoogleTestLoader
 + (void)load {
     NSBundle *bundle = [NSBundle bundleForClass:self];
     [[NSNotificationCenter defaultCenter] addObserverForName:NSBundleDidLoadNotification object:bundle queue:nil usingBlock:^(NSNotification *notification) {
-        [self registerTestClasses];
+        [GoogleTestCase initialize];
```

Comment:
Hello,

I had the bug where registerTestClasses was called twice, this is how I fixed it:

        NSArray* loadedClasses = [notification.userInfo objectForKey:NSLoadedClasses];
        
        if( loadedClasses != nil ) {
            if( [loadedClasses indexOfObject:@"GoogleTestLoader"] != NSNotFound) {
                [self registerTestClasses];
            }
        }